test/assets/unrunnableTestCases.csv

```diff
@@ -2656,77 +2636,64 @@ K2-DirectConElemWhitespace-5,Error: Not implemented: only module imports, namesp
 K2-DirectConElemWhitespace-6,Error: Not implemented: only module imports, namespace declarations, and function declarations are implemented in XQuery modules
 K2-DirectConElemWhitespace-7,Error: Not implemented: only module imports, namespace declarations, and function declarations are implemented in XQuery modules
 K2-DirectConElemWhitespace-8,Error: Not implemented: only module imports, namespace declarations, and function declarations are implemented in XQuery modules
-eqname-002,Error: Not implemented: references to variables with a namespace URI or a prefix.
 eqname-004,Error: XPST0081: The type :integer could not be found.
 eqname-006,Error: No selector counterpart for: extensionExpr.
 eqname-007,Error: XPST0003: Unable to parse: "         declare decimal-format Q{http://www.example.com/ns}format grouping-separator="'";          <a xmlns:ex="http://www.example.com/ns">{format-number(1e9, "#'###'###'##0.00", 'ex:format')}</a>       ". Expected " ", "(:", "\n", "\r", or "\t" but "=" found.          declare decimal-format Q{http://www.example.com/ns}format grouping-separator[Error is around here]="'";          <a xmlns:ex="http://www.example.com/ns">{format-number(1e9, "#'###'###'##0.00", 'ex:format')}</a>
 eqname-008,AssertionError: Expected XPath <out>{ (<a xmlns:ex="http://www.example.com/ns"><ex:b>93.7</ex:b></a>) /Q{http://www&#x2e;example&#x2E;com/ns}b }</out> to resolve to the given XML. Expected <out/> to equal <out><ex:b xmlns:ex="http://www.example.com/ns">93.7</ex:b></out>
-eqname-015,Error: Not implemented: references to variables with a namespace URI or a prefix.
 eqname-017,AssertionError: Expected XPath string-join(<a foo="3" bar="5" xml:space="preserve"/> / @Q{http://www.w3.org/XML/1998/namespace}*, '.') to resolve to "preserve": expected false to be true
-eqname-024,Error: Not implemented: references to variables with a namespace URI or a prefix.
+eqname-024,Error: XPST0008, The variable x is not in scope.
```

Comment:
This seems to indicate some parse errors concerning white space. (removing white spaces between foo and bar fixes prevents the error)

The test case:
```xml
<test-case name="eqname-024">
   <description>Tests that the whitespace is properly collapsed by comparing single-ws with multi-ws URIs</description>
   <created by="Abel Braaksma" on="2016-06-27"/>
   <dependency type="spec" value="XP30+ XQ30+"/>
   <test>for $Q{ urn:foo bar }x in 1 to 5 return $Q{urn:foo   bar}x + $Q{urn:foo bar}x</test>
   <result>
      <assert-string-value>2 4 6 8 10</assert-string-value>
   </result>
</test-case>```

Reply:
Whitespace normalization in namespace uris is a whole can of worms we haven't even started to open, so I'm fine with leaving this as it is